app/views/content/welcome/jacks-story/_article.html.erb

```diff
@@ -1,4 +1,4 @@
-<article class="story">
+<article class="case-study">
   <p>
     Jack is a PE teacher from Leeds, with a passion for opening up conversations
```

Comment:
Additional changes for Jack's story (can't suggest directly):

Line 16 - remove full stop after **...possible?'** as double punctuation

Line 28 - replace ampersand with "and"